src/Symfony/Component/Serializer/Encoder/JsonDecode.php

```diff
@@ -95,8 +95,11 @@ public function decode($data, $format, array $context = array())
             $decodedData = json_decode($data, $associative, $recursionDepth);
         }
 
-        if (JSON_ERROR_NONE !== $this->lastError = json_last_error()) {
-            throw new UnexpectedValueException(JsonEncoder::getLastErrorMessage());
+        if (false === $decodedData) {
+            $this->lastError = json_last_error();
+            throw new UnexpectedValueException(
```

Comment:
should be on one line according to our CS policy

Reply:
fixed